resources/services/mq/brokers.go

```diff
@@ -45,29 +50,30 @@ func MqBrokers() *schema.Table {
 			},
 			{
 				Name:        "arn",
-				Description: "The Amazon Resource Name (ARN) of the broker.",
+				Description: "The broker's Amazon Resource Name (ARN).",
 				Type:        schema.TypeString,
 				Resolver:    schema.PathResolver("BrokerArn"),
 			},
 			{
-				Name:        "broker_id",
+				Name:        "id",
```

Comment:
this is a breaking change, can you verify that know policies uses this field?

Reply:
no policies use this column